### ISSUE TITLE
Feature Discussion: Automated install by running menu.py on media creation. 

### ISSUE THREAD

**Issue by ajd394** [fcaviggia/hardened-centos7-kickstart#15]
Unless i am missing something, I feel that the media.py is only helpful for attended installations because you need to manually enter parameters on boot. I think it would be better if there were also an option to run menu.py before installation and include a config file that would allow for an automated installation. Thoughts?

**Comment by ajd394** [comment]
@fcaviggia what do you think about this idea?

**Comment by fcaviggia** [comment]
Yeah, I've done a bypass kickstart already already using packer - I'll look into adding a bypass if a named file is included in config/hardening directory. Right now you can always just replace 'hardened-centos.cfg' with your own kickstart script to make a build that does a single purpose installation.

**Comment by ajd394** [comment]
@fcaviggia I'm looking into using packer also so that i can have a vagrant vm that is similar to the production install via iso. can you share your bypass kickstart and packer json?

**Comment by fcaviggia** [comment]
Unfortunately, I have to get MITRE legal to look over the scripts before I release them. I think I have to change the legal banners at a minimum.
